### ISSUE TITLE
Plugin registers a new account on every run

### ISSUE THREAD

**Issue by jameysharp** [dokku/dokku-letsencrypt#240]
## Description of problem

The plugin registers a new account at the Let's Encrypt server every time the `letsencrypt:enable` command runs, which makes it way too easy to hit the rate limit of registering 10 accounts within 3 hours.

This happens because the `config_hash` includes `$acme_port`, which is different every time the `letsencrypt:enable` command runs. I believe the bug was introduced in 9ed59d8fe6ff983a5925b9b10b11184f12b25d4c.

### How reproducible

Reliably reproducible

### Steps to Reproduce

1. Run `dokku letsencrypt:enable $app` for an app which has already been enabled.
2. In the output, shortly after "Getting letsencrypt certificate for $app...", observe the message "No key found for account <...>. Generating a P256 key."

## Environment Information

I encountered this bug in release 0.12.1.

My current workaround is that I've downgraded to 0.10.1 as the last version which could use my previous account registration information.

**Comment by boredland** [comment]
Hi! Due to this, I can't revoke certificates at the moment:
```
=====> Revoke letsencrypt certificate from xx...
-----> Revoking letsencrypt certificate for xx...
        - Domain 'xxx'
       2021/10/04 08:42:34 No key found for account xxx. Generating a P256 key.
       2021/10/04 08:42:34 Saved key to /certs/accounts/acme-v02.api.letsencrypt.org/xxx.key
       2021/10/04 08:42:50 Account xxx is not registered. Use 'run' to register a new account.
-----> Certificate revocation failed (code )!
       done
```

**Comment by berinhard** [comment]
@boredland any change that your PR #244 fixed this issue? I'm facing it too.

**Comment by achainan** [comment]
Is this fixed? I'm running into the same issue. I am unable to renew my domains. I tried downgrading as @jameysharp suggested and it still doesn't work.

**Comment by boredland** [comment]
Hi! My try to fix could work, I had to move away from ad-hoc certificates because I ran into the rate limits and so I currently don't use & not able to test this anymore, sorry...

**Comment by jameysharp** [comment]
I don't think #244 is quite right, because `tr -d` removes any instance of each specified character. In this case it would remove all spaces, all colons, and up to five different digits.

I think the pull request I just opened, #250, should work reliably. But I'm not sure how to try it on my server. I don't know how to switch an already-installed plugin to a different branch or committish. The things I've tried so far didn't do it.

@josegonzalez, could you review and test my pull request?

**Comment by ikalkov** [comment]
We face the same issue in 0.12.1

**Comment by josegonzalez** [comment]
I've merged #250. Can folks upgrade to 0.13.0 and verify it doesn't recreate accounts?

**Comment by boredland** [comment]
> I've merged #250. Can folks upgrade to 0.13.0 and verify it doesn't recreate accounts?

Will try tomorrow
